Makefile

```diff
@@ -143,9 +146,9 @@ run: manifests generate fmt vet ## Run a controller from your host.
 docker-build: test ## Build docker image with the manager.
 ifneq (, $(PEERPODS))
 	@echo PEERPODS is enabled
-	docker build -t ${IMG} -f Dockerfile.peerpods .
+	docker build --build-arg http_proxy=$(http_proxy) --build-arg https_proxy=$(https_proxy) -t ${IMG} -f Dockerfile.peerpods .
```

Comment:
is there a reason why this approach was chosen over the better alternative which just setting the `~/.docker/config.json`?

Reply:
@mythi I have the proxies at the config but seems that when building the dockerfile is still failing without those arguments

Reply:
I don't prefer this approach. It'd be good to understand *why* the best option does not work